KMaina/Fast-Food-App
# #160219322 Landing Page for The Site

#### What does this PR do?

It shows the landing page for the website

#### Description of Task to be completed?

A user should be able to

- Log in if they already have an account with the site.

- Sign up and create an account with the site.

#### How should this be manually tested?

- Clone the repo onto your local machine.

- Open a terminal and run `cd Fast-Food-App`

- run `git checkout -b ft-landing-page-design-160219322`

- run `git pull https://github.com/KMaina/Fast-Food-App.git ft-landing-page-design-160219322`

- In the folder where you cloned the repo, you should open the UI folder.

- Open the index.html file to see it in your browser for review.

#### Any background context you want to provide?

This is simply the UI design, no functionality has been built in yet.

#### What are the relevant pivotal tracker stories?

#160219322
